src/lib.rs

```diff
@@ -1,5 +1,7 @@
-// DOCS
```

Comment:
Check this: https://github.com/untitaker/rust-atomicwrites/blob/a0db3c6ad80b834c50ee26fda51afd8c3d3f24d0/Makefile#L14

I believe there are better ways to achieve this nowadays

Reply:
So my request is, either restore this behavior (I think there was a crate for that) or clean up the makefile to remove this useless logic :)

Reply:
I changed it back (with a more descriptive replacement text), PTAL.